src/tests/pools/D3MAavePool.t.sol

```diff
@@ -249,28 +247,30 @@ contract D3MAavePoolTest is D3MPoolBaseTest {
         assertEq(D3MAavePool(d3mTestPool).redeemable(), address(adai));
     }
 
-    function test_transfer_adai() public {
+    function test_exit_adai() public {
         uint256 tokens = adai.totalSupply();
         adai.transfer(d3mTestPool, tokens);
         assertEq(adai.balanceOf(address(this)), 0);
         assertEq(adai.balanceOf(d3mTestPool), tokens);
 
-        D3MAavePool(d3mTestPool).file("hub", address(this));
-        D3MAavePool(d3mTestPool).transfer(address(this), tokens);
+        end.setArt(tokens);
+        vm.prank(hub);
+        D3MAavePool(d3mTestPool).exit(address(this), tokens);
```

Comment:
Am I understanding this correctly? `vm.prank` do not change the value of `address(this)` it only changes `msg.sender`?

Reply:
Right, that's how I understand it.